next/api/resources/retired.py

```diff
@@ -0,0 +1,20 @@
+from flask import Flask, request, send_file
```

Comment:
How does adding a route `/dashboard/experiment/<exp_uid>/retire` to `dashboard.py` sound?

Otherwise, I can just move that resource to `resources/experiment.py`.